crates/bindings-csharp/Runtime/Internal/Module.cs

```diff
@@ -269,17 +287,15 @@ public static Buffer __describe_module__()
             // We need this explicit cast here to make `ToBytes` understand the types correctly.
             RawModuleDef versioned = new RawModuleDef.V8BackCompat(moduleDef);
             var moduleBytes = IStructuralReadWrite.ToBytes(new RawModuleDef.BSATN(), versioned);
-            var res = FFI._buffer_alloc(moduleBytes, (uint)moduleBytes.Length);
-            return res;
+            description.Write(moduleBytes);
         }
         catch (Exception e)
         {
             Runtime.Log($"Error while describing the module: {e}", Runtime.LogLevel.Error);
-            return Buffer.INVALID;
         }
     }
 
-    public static Buffer __call_reducer__(
+    public static short __call_reducer__(
```

Comment:
Maybe change the return value to the `Errno` so that we could return typechecked enum directly?